lib/compilers/codon.ts

```diff
@@ -0,0 +1,31 @@
+import type {ParseFiltersAndOutputOptions} from '../../types/features/filters.interfaces.js';
```

Comment:
Can you add the copyright banner please? You can copy it from another file (making sure the date is correct; and the authors are either you or "Compiler Explorer Authors")

Reply:
Sure, added!